dev-docs/source/GettingStarted.rst

```diff
@@ -144,3 +144,48 @@ OSX
 * It can be found at `/Volumes/inst$`
 
 **NB** the address in step 2 sometimes changes - if it does not work, replace `80` with `55` or `3`.
+
+Linux
+------
+1. Install packages:
+
+* ``autofs``
+* ``smbfs`` - Ubuntu 12.04
+* ``cifs-utils`` - Ubuntu 14.04
+
+2. Create an ``/archive.creds`` file in the root directory containing this, filling in the relevant details:
```

Comment:
Could you also note that this should only be done with full disk encryption enabled (or `archive.creds` stored somewhere secure, e.g. an encrypted home directory).
This is required to ensure passwords are safe at rest.

The file should also have permissions of `-r--------` (`chmod 400`).